curriculum/challenges/english/25-front-end-development/workshop-greeting-card/67333a3a400b0a3c68854bcc.md

```diff
@@ -0,0 +1,125 @@
+---
+id: 67333a3a400b0a3c68854bcc
+title: Step 21
+challengeType: 0
+dashedName: step-21
+---
+
+# --description--
+
+Add to the `.card-links a` selector a `transition` property with a value of `background-color 0.3s ease`.
+
+# --hints--
+
+The `.card-links a` selector should have a `transition` property set to `background-color 0.3s ease`.
+
+```js
+assert.strictEqual(
```

Comment:
Same here, not able to pass because the value is `background-color 0.3s`

Reply:
can you check again? I did some changes to the two tests